internal/scan-manager/webapp/src/main/java/org/wso2/security/tools/scanmanager/webapp/Application.java

```diff
@@ -19,14 +19,31 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+import static org.springframework.security.config.Customizer.withDefaults;
 
 /**
  * The main class that starts the spring boot application.
  */
+@Configuration
+@EnableWebSecurity
 @SpringBootApplication
-public class Application {
+public class Application extends WebSecurityConfigurerAdapter {
 
     public static void main(String[] args) {
         SpringApplication.run(Application.class, args);
     }
+
+    @Override
+    public void configure(HttpSecurity http) throws Exception {
+        http.csrf().disable()
```

Comment:
is `http.csrf().disable()`intended. Do we handle CSRF issues in a different manner?

Reply:
This is fixed with the new commit.